test/cmdlineTests/transient_storage_layout_smoke_two_contracts/input.json

```diff
@@ -0,0 +1,19 @@
+{
+	"language": "Solidity",
+	"sources": {
+		"fileA": {
+			"content": "//SPDX-License-Identifier: GPL-3.0\npragma solidity >=0.0;\ncontract A { }"
+		},
+		"fileB": {
+			"content": "//SPDX-License-Identifier: GPL-3.0\npragma solidity >=0.0;\ncontract B { uint transient x; uint transient y; }"
+		}
+	},
+	"settings": {
+		"outputSelection": {
+			"fileA": {
+				"A": ["transientStorageLayout"],
+				"B": ["transientStorageLayout"]
+			}
```

Comment:
`B`'s storage layout is missing from the output. Note that it's defined in `fileB`, not `fileA` :)

Reply:
Fixed.